app/src/main/res/layout/dialog_filter.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingTop="16dp">
+
+    <EditText
+        android:id="@+id/phraseEditText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingEnd="24dp"
+        android:paddingStart="24dp"
+        android:hint="@string/filter_add_description"
+        app:layout_constraintTop_toTopOf="parent"
+        />
+        <!-- Mimic AlertDialog positive/negative/neutral layout, but with our padding -->
```

Comment:
I don't think this is a good idea if not absolutely necessary, can we stay with the default layout please?

Reply:
Ah, it's not necessary after the `AlertDialog` import fix